airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/DatabaseConfigPersistenceTest.java

```diff
@@ -50,29 +55,75 @@
  * methods.
  */
 @SuppressWarnings({"PMD.SignatureDeclareThrowsException", "PMD.ShortVariable", "PMD.JUnitTestsShouldIncludeAssert"})
-class DatabaseConfigPersistenceTest extends BaseDatabaseConfigPersistenceTest {
+class DatabaseConfigPersistenceTest extends BaseConfigDatabaseTest {
+
+  public static final String DEFAULT_PROTOCOL_VERSION = "0.2.0";
+  protected static final StandardSourceDefinition SOURCE_GITHUB = new StandardSourceDefinition()
```

Comment:
all of this noise is copied from `BaseDatabaseConfigPersistenceTest`. It will all get removed in the next PR which gets rid of `DatabaseConfigPersistence` entirely.